cugraph/traversal/SSSP.ipynb

```diff
@@ -237,8 +236,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Call cugraph.sssp to get the distances from vertex 1:\n",
-    "df = cugraph.sssp(G, 1)"
+    "# Call cugraph.sssp to get the distances from vertex 0:\n",
+    "df = cugraph.sssp(G, 0)"
```

Comment:
The dataset, the text content and the graph drawing are all one-based indexed. 0 is not a valid input here. Renumbering should be set to `True`